src/parser.js

```diff
@@ -1392,9 +1430,11 @@ export default class Parser {
     }
 
     if (handledSchema) {
+      // can we find the schema?
```

Comment:
Needs to be fixed, in the case that the response/request was referenced `this.path` will be incorrect from dereferenced document. I think it will be a hassle to solve this, we need to deref document except the schemas. Workaround could be attempt to fix it and fallback to derefed schema.